fsc/hdf5_io/_version.py

```diff
@@ -2,10 +2,7 @@
 Defines the module's version, read from the version.txt file.
 """
 
-import os
+import pathlib
 
-with open(
-    os.path.join(os.path.dirname(os.path.abspath(__file__)), 'version.txt'),
-    'r'
-) as f:
+with open(pathlib.Path(__file__).parent.resolve() / 'version.txt', 'r') as f:
```

Comment:
should work, but .resolve().parent is safer

Reply:
nvm, always works

Reply:
How so?

Reply:
its a file, which always has a parent

Reply:
for folders you might run out of parent before resolve, but never tested it, just a hunch

Reply:
Ah, recommendation is here: https://docs.python.org/3.5/library/pathlib.html#pathlib.PurePath.parent

Reply:
Yes, so that was it, but we can leave it here, just for the future

Reply:
Well, better to build a good habit.